Expand contributors on project card

jngong opened the pull request:
Closes #93 Project card: 'expand contributors'

Discussed with Nev to use an icon instead of ellipses to signal expanded contributors
- Had to add nodemon as a dev dependency to start the server 
- Added font awesome libraries as dependencies 
- Added new class name for styling icon 
- Added state attribute to track whether or not contributors are expanded
- Edited getContributors method logic to map based on state 
- Added method to render expansion icon based on length of the list

@nurmerey @partiallymisplaced 
Let me know if there is anything that needs to be changed. Looking forward to your feedback!

**Screenshot -- When contributors are not expanded**
![Screen Shot 2020-10-01 at 6 17 29 PM](https://user-images.githubusercontent.com/34215359/94869483-675d7580-0413-11eb-88da-22b71e4e5081.png)

**Screenshot -- When contributors are
![Screen Shot 2020-10-01 at 6 17 36 PM](https://user-images.githubusercontent.com/34215359/94869532-8825cb00-0413-11eb-99f1-808124547047.png)
 expanded**

partiallymisplaced commented on src/components/ProjectCard.tsx:
```diff
@@ -70,10 +72,29 @@ const renderButtons = (project: Project, loginLink: any) => {
+
+  if (contributorsLength > 6) {
+    return <div onClick={handleContributorsClick} className="card-contributor-icon-expand">
+      <FontAwesomeIcon icon={faArrowsAltV} />
```
Instead of a single arrows icon, this should be two separate elements - [caret down](https://fontawesome.com/icons/caret-down?style=solid) when contributors are collapsed, and [caret up](https://fontawesome.com/icons/caret-up?style=solid) when contributors are expanded 😊

jngong replied:
Thank you @partiallymisplaced ! I've updated my code to use the two separate icons for expanding and collapsing. Here are some screenshots:
<img width="379" alt="Screen Shot 2020-11-01 at 1 44 25 PM" src="https://user-images.githubusercontent.com/34215359/97811663-06130700-1c4a-11eb-9315-a31048d849cc.png">
<img width="392" alt="Screen Shot 2020-11-01 at 1 44 19 PM" src="https://user-images.githubusercontent.com/34215359/97811664-07dcca80-1c4a-11eb-99b3-7c2c90c07e19.png">

nurmerey commented:
@ctran14 this one addressed #93 .

nurmerey commented:
Hi @jngong! It looks like you are very close to finishing this PR work, let us know if you need any help.

jngong commented:
Thanks for the feedback @numerey and @partiallymisplaced! Sorry I missed this. I will try to get to addressing the comments later in the week :) Should be a quick fix!

jngong commented:
@nurmerey and @partiallymisplaced I've made the change requested and also resolved merge conflicts with master. Thank you!

partiallymisplaced reviewed (approved):
Awesome, thank you!!! ✨👏
